docs/02-app/01-building-your-application/11-upgrading/05-from-create-react-app.mdx

```diff
@@ -64,55 +64,13 @@ Create a `next.config.mjs` at the root of your project. This file will hold your
 /** @type {import('next').NextConfig} */
 const nextConfig = {
   output: 'export', // Outputs a Single-Page Application (SPA).
-  distDir: './dist', // Changes the build output directory to `./dist/`.
+  distDir: './build', // Changes the build output directory to `./dist`.
 }
 
 export default nextConfig
 ```
 
-### Step 3: Update TypeScript Configuration
-
-If you're using TypeScript, you need to update your `tsconfig.json` file with the following changes to make it compatible with Next.js:
-
-```json filename="tsconfig.json"
-{
-  "compilerOptions": {
-    "target": "es5",
-    "lib": ["dom", "dom.iterable", "esnext"],
-    "allowJs": true,
-    "skipLibCheck": true,
-    "strict": false,
-    "forceConsistentCasingInFileNames": true,
-    "noEmit": true,
-    "esModuleInterop": true,
-    "module": "esnext",
-    "moduleResolution": "node",
-    "resolveJsonModule": true,
-    "isolatedModules": true,
-    "jsx": "preserve",
-    "baseUrl": ".",
-    "incremental": true,
-    "plugins": [
-      {
-        "name": "next"
-      }
-    ],
-    "strictNullChecks": true
-  },
-  "include": [
-    "next-env.d.ts",
-    "**/*.ts",
-    "**/*.tsx",
-    ".next/types/**/*.ts",
-    "./dist/types/**/*.ts"
-  ],
-  "exclude": ["node_modules"]
-}
-```
-
-You can find more information about configuring TypeScript on the [Next.js docs](/docs/app/building-your-application/configuring/typescript#typescript-plugin).
-
```

Comment:
The recommended `tsconfig.json` file configuration for Next.js often changes. Moreover the user might have some custom TypeScript configuration that they might want to keep (e.g., `noUncheckedIndexedAccess`).

Instead of asking them to manually update it, we will rely on the TypeScript auto-configuration of the `next dev` command in a later step.